docs/how-to-guides/new-user-guides/authentication-permissions-and-global-configuration/authentication-config/configure-azure-ad.md

```diff
@@ -113,7 +113,21 @@ Ensure that you set Application permissions, and *not* Delegated permissions. Ot
 
 :::note
 
-Rancher doesn't validate the permissions you grant to the app in Azure. We only support the use of the `Directory.Read.All` application permission.
+Rancher doesn't validate the permissions you grant to the app in Azure. This means you are free to try any permissions you want, just ensure that they are sufficient for Rancher to work with AD users and groups.
+
+Specifically, Rancher needs to do the following:
+- Get a user
+- List all users
+- List groups of which a given user is a member
+- Get a group
+- List all groups
```

Comment:
```suggestion
- Get a user.
- List all users.
- List groups of which a given user is a member.
- Get a group.
- List all groups.
```

Reply:
Out of curiosity, why do we need periods at ends of list items? These aren't sentences...
Also, should I add them below where I list example permission combinations?

Reply:
The style guide we use specifies using a period at the end of each list item if at least one of the list items can be read as a sentence, even if they're very short.